site/_guides/gateway-api.md

```diff
@@ -0,0 +1,118 @@
+- __Service Operator__: The Service Operator is responsible for defining application configuration and service
+  composition. They will interact with xRoute resources and other typical Kubernetes resources.
+
+Gateway API contains three primary resources:
+
+- __GatewayClass__: Defines a set of gateways with a common configuration and behavior.
+- __Gateway__: Requests a point where traffic can be translated to a Service within the cluster.
+- __xRoutes__: Describes how traffic coming via the Gateway maps to the Services.
+
+Resources are meant to align with personas. For example, a platform operator will create a Gateway, so a developer can
+expose an HTTP application using an HTTPRoute resource.
+
+### Prerequisites
+The following prerequisites must be met before using Gateway API with Contour:
+
+- A working [Kubernetes][2] cluster. Refer to the [compatibility matrix][3] for cluster version requirements.
+- The [kubectl][4] command-line tool, installed and configured to access your cluster.
+
+### Using Gateway API with Contour Operator
+
+The preferred method for using Gateway API is with [Contour Operator][5]. This is because the GatewayClass and Gateway
+resources should be managed by active controllers that are implemented by the operator. Refer to the [contour][6] and
+[operator][7] designs for additional background on the gateway API implementation.
+
+Run the operator:
+```shell
+$ kubectl apply -f https://projectcontour.io/quickstart/operator.yaml
+```
+This command creates:
+
+- Namespace `contour-operator` to run the operator.
+- Operator and Contour CRDs.
+- Operator RBAC resources for the operator.
+- A Deployment to manage the operator.
+- A Service to front-end the operator’s metrics endpoint.
+
+Create the Gateway API resources:
+```shell
+$ kubectl apply -f https://projectcontour.io/quickstart/gateway.yaml
+```
+This command creates:
+
+- Namespace `projectcontour` to run the Gateway and child resources, i.e. Envoy DaemonSet.
+- A Contour custom resource named `contour-gateway-sample` in the operator's namespace. This resource exposes
+  infrastructure-specific configuration and is referenced by the GatewayClass.
+- A GatewayClass named `sample-gatewayclass` that abstracts the infrastructure-specific configuration from Gateways.
+- A Gateway named `contour` in namespace `projectcontour`. This gateway will serve the test application through routing
+  rules deployed in the next step.
+
+### Testing the Gateway API
+
+Run the test application:
+```shell
+$ kubectl apply -f https://projectcontour.io/quickstart/kuard.yaml
+```
+This command creates:
+
+- A Deployment named `kuard` in namespace `projectcontour` to run kuard as the test application.
+- A Service named `kuard` in namespace `projectcontour` to expose the kuard application on TCP port 80.
+- An HTTPRoute named `kuard` in namespace `projectcontour` to route requests for "*.projectcontour.io" to the kuard
+  service.
```

Comment:
Also not a big deal, but consider installing the workload in a namespace other than `projectcontour` since it usually wouldn't go in there.

Reply:
@skriss "SameNamespace" is the [default](https://github.com/kubernetes-sigs/gateway-api/blob/master/apis/v1alpha1/shared_types.go#L65) for a route. I wanted to stick with defaults as much as possible to keep things simple.